presto-docs/src/main/sphinx/aws/emr.rst

```diff
@@ -0,0 +1,28 @@
+=============
+Presto on EMR
+=============
+
+TODO: Add an intro and links to appropriate presto-admin docs
```

Comment:
I dint know if you were already adding anything here , @petroav . The EMR resizing should probably be a new section here.